README.md

```diff
@@ -54,16 +55,27 @@ In this step, we'll view the same HSSD scene on the Quest VR headset. No HITL se
 5. Put on your headset. The app may already be running. If not, or if you encounter issues, hit the Quest home button and it should let you quit the `siro_hitl_vr_client` program. After you quit, you can usually find the icon for the program in your list of recently-run programs. You should re-launch it.
 6. When in the scene, press and hold forward on either VR controller thumbstick to aim for a teleportation destination, then release the thumbstick to teleport.
```

Comment:
You should fix these movement instructions. I wrote them when the controls were broken on my local machine.